debian/control

```diff
@@ -39,6 +39,8 @@ Vcs-Browser: https://github.com/ArcticaProject/nx-libs/
 Package: nx-x11-common
 Architecture: all
 Multi-Arch: foreign
+Pre-Depends:
+ dpkg (>= 1.15.7.2),
```

Comment:
Also unrelated change in debian/control. Drop this commit. Pre-Depends is not necessary here.